src/App.js

```diff
@@ -26,6 +27,7 @@ const App = () => (
       </Route>
     </Switch>
   </Router>
+  </React.StrictMode>
 );
 
 export default App;
```

Comment:
> ***NOTE: This change is only applicable to the import statement. I'm using this inline comment to be able to add a review. please refer to the import statement.***

- Hey Team, great job with the test, however, the test is failing due to how you've wrongly imported the calculator components. 
- As you used a lowercase letter for the `c` which you've imported as an Uppercase `C`. For this reason, your test are not passing. Please kindly fix this issue to make the test pass.

`import Calculator from './components/calculator';
`